spec/models/customer_spec.rb

```diff
@@ -32,7 +32,7 @@
     end
 
     it 'has unique email' do
-      FactoryBot.create(:customer, email: 'email@unique.com') 
+      FactoryBot.create(:customer, email: 'email@unique.com')
```

Comment:
Can be handled by shoulda matchers
`it { is_expected.to validate_uniqueness_of(:email) }`